src/kili/services/export/format/coco/__init__.py

```diff
@@ -313,7 +315,7 @@ def _get_coco_image_annotations(
                 id=annotation_j,
                 image_id=asset["id"],
                 category_id=cat_kili_id_to_coco_id[categories[0]["name"]],
-                bbox=[int(np.min(p_x)), int(np.min(p_y)), int(np.max(p_x)), int(np.max(p_y))],
+                bbox=[int(x_min), int(y_min), int(_w_), int(_h_)],
```

Comment:
to make sure this does not happen again, can we add a unit test? if can be worth it to extract a method `_get_coco_bbox_from_kili_bpoly`